pkg/utils/etcdutil/etcdutil.go

```diff
@@ -707,7 +707,6 @@ func (lw *LoopWatcher) watch(ctx context.Context, revision int64) (nextRevision
 	}()
 	ticker := time.NewTicker(RequestProgressInterval)
 	defer ticker.Stop()
-	lastReceivedResponseTime := time.Now()
```

Comment:
Don't need to delete the same line in `leadership.go`?

Reply:
I got it, API server never meets this problem.

Reply:
No, `leadership` has other healthy check `if time.Since(lastReceivedResponseTime) > unhealthyTimeout `